README.md

```diff
@@ -559,3 +559,67 @@ To run a development `iex` REPL with all code loaded:
 ```bash
 iex -S mix run --no-start
 ```
+
+# Working with API Spec's
```

Comment:
We have described it better in the README of `gh-pages` branch. Probably here we should just point to that README